pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.444</version>
+        <version>1.609.1</version>
```

Comment:
This PR introduces use of `SimpleBuildWrapper` which is  available in core only since 1.599, and 1.609.1 is the first LTS release containing it. See https://wiki.jenkins-ci.org/display/JENKINS/Plugin+Report+Card for why this is done.